internal/load/README.md

```diff
@@ -289,6 +290,7 @@ type ArgumentList struct {
 	ContainerDiscoveryDir   string `default:"flexContainerDiscovery/" help:"Set directory of auto discovery config files"`
 	ContainerDiscovery      bool   `default:"false" help:"Enable container auto discovery"`
 	ContainerDiscoveryMulti bool   `default:"false" help:"Allow a container to be matched multiple times"`
+	ContainerDump           bool   `default:"false" help:"Dump all containers, useful for debugging"`
```

Comment:
Is this field being used in this PR?

Reply:
This is documentation. It's auto generated so it seems it was missing from the readme